src/providers/WorkflowCore.Persistence.EntityFramework/Services/EntityFrameworkPersistenceProvider.cs

```diff
@@ -16,12 +16,14 @@ public class EntityFrameworkPersistenceProvider : IPersistenceProvider
         private readonly bool _canCreateDB;
         private readonly bool _canMigrateDB;
         private readonly IWorkflowDbContextFactory _contextFactory;
+        private readonly ModelConverterService _modelConverterService;
 
         public bool SupportsScheduledCommands => true;
 
-        public EntityFrameworkPersistenceProvider(IWorkflowDbContextFactory contextFactory, bool canCreateDB, bool canMigrateDB)
+        public EntityFrameworkPersistenceProvider(IWorkflowDbContextFactory contextFactory, ModelConverterService modelConverterService, bool canCreateDB, bool canMigrateDB)
```

Comment:
We should depend on abstractions rather than concrete implementations, this should be an interface